tests/yast2_gui/yast2_control_center.pm

```diff
@@ -40,9 +40,6 @@ sub search {
         send_key 'backspace';
     }
     wait_screen_change { type_string $name; } if $name;
-    # After typing some icons get detected before it's filetered
-    # Adding extra sync point before trying to click
-    wait_still_screen 3;
```

Comment:
This call was a hack to stabilize the test execution, and we paid with execution time for it.
On SLES it started to fail now: https://openqa.suse.de/tests/4189369#

Reply:
yes, @b10n1k also addressed me with this. Have you looked into the suggestions I gave? E.g. basically follow what DimStar77 has written in the commit message about the needle design. Maybe you already solved this together with DimStar77 over another channel …

Reply:
Nobody reached out to me, but essentially:

* Delete the needle that matched here: https://openqa.suse.de/tests/4189369#step/yast2_control_center/39
* Create a new needle after the search is complete (like in https://openqa.suse.de/tests/4189369#step/yast2_control_center/41); include the category title 'Software' (the blue bar) into the needle; so it can only match if and when 'media check' is the item top left in the search window (aka search completed).S et click coordinates to make sure you click at the right spot ('center' of the needle might not be good enough)

Reply:
Yep, @b10n1k did this exercise already: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1370/diffs 
I've checked that we don't run this test suite for quarterly updates for SP1, so should be fine.
Thanks!